src/material/slider/slider.spec.ts

```diff
@@ -438,6 +438,13 @@ describe('MatSlider', () => {
         expect(sliderInstance.percent).toBe(1);
       },
     );
+
+    it('should calculate percentage propertly when min,max,value are zero', () => {
+      sliderInstance.value = 0;
+      sliderInstance.min = 0;
+      sliderInstance.max = 0;
+      expect(sliderInstance.percent).toBe(1);
```

Comment:
This seems weird to me. Shouldn't the `percent` be 0 if everything everything is 0?

Reply:
As I understand percent is how the value is proportional between min and max , so if you have value=0 and the min and max value=0 as well the percent should be 1 not 0, it's like to have value=2 and max=2 as well

Reply:
I'm thinking about it in terms of what would show up in the UI. As a user, it would be weird if the value was zero but the slider was full.